.eslintrc.json

```diff
@@ -34,6 +34,7 @@
       "singleQuote": true
     }],
     "func-names": ["warn", "as-needed"],
-    "jsx-a11y/label-has-for": 0
+    "jsx-a11y/label-has-for": 0,
+    "import/prefer-default-export": 0
```

Comment:
IMO this isn't needed. Mostly for util files, I don't want to add the first one as a default export if I think there may be more functions added later.